aws_lambda_powertools/utilities/batch/exceptions.py

```diff
@@ -1,7 +1,23 @@
 """
 Batch processing exceptions
 """
+import traceback
 
 
 class SQSBatchProcessingError(Exception):
     """When at least one message within a batch could not be processed"""
+
+    def __init__(self, msg="", child_exceptions=()):
+        super().__init__(msg)
+        self.msg = msg
+        self.child_exceptions = child_exceptions
+
+    def __str__(self):
+        parent_exception_str = super(SQSBatchProcessingError, self).__str__()
```

Comment:
Could you add a comment as to why we're doing this? Maybe a ref to the issue, so it's easier to remember why